src/tck/resources/META-INF/vertx/json-mappers.properties

```diff
@@ -0,0 +1,10 @@
+io.vertx.codegen.testmodel.MyPojoToInteger.serializer=io.vertx.codegen.testmodel.JsonMapperTCK#serializeMyPojoToInteger
+io.vertx.codegen.testmodel.MyPojoToInteger.deserializer=io.vertx.codegen.testmodel.JsonMapperTCK#deserializeMyPojoToInteger
+io.vertx.codegen.testmodel.MyPojoToJsonObject.serializer=io.vertx.codegen.testmodel.JsonMapperTCK#serializeMyPojoToJsonObject
+io.vertx.codegen.testmodel.MyPojoToJsonObject.deserializer=io.vertx.codegen.testmodel.JsonMapperTCK#deserializeMyPojoToJsonObject
+io.vertx.codegen.testmodel.MyPojoToJsonArray.serializer=io.vertx.codegen.testmodel.JsonMapperTCK#serializeMyPojoToJsonArray
+io.vertx.codegen.testmodel.MyPojoToJsonArray.deserializer=io.vertx.codegen.testmodel.JsonMapperTCK#deserializeMyPojoToJsonArray
+java.time.ZonedDateTime.serializer=io.vertx.codegen.testmodel.JsonMapperTCK#serializeZoneDateTime
```

Comment:
Where do you want to ship the serializers/deserializers for `ZonedDateTime` and `Locale`? Directly from vertx-core?

Reply:
I don't know yet